src/components/Todolist.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { completedTodo, deleteTodo } from "../redux/todo";
+
+export default function TodoList(): ReactElement {
+  const dispatch = useDispatch();
+  const tasks = useSelector(
+    (state: {
+      todo: {
+        value: {
+          id: string;
+          text: string;
+          isCompleted: boolean;
+        }[];
+      };
+    }) => state.todo.value
+  );
+  const todoTasks = tasks
+    .filter((task, index) => index > 0)
+    .map((task) => (
```

Comment:
- Your task variable passed in the filter methods shown below
```suggestion
    .filter((task, index) => index > 0)
    .map((task) => (
```

Displays this error ```'task' is declared but its value is never read.ts(6133)
(parameter) task: {
    id: string;
    text: string;
    isCompleted: boolean;
}```

I would prefer that you add the underscore before the word to remove the red lines.
```suggestion
    .filter((_task, index) => index > 0)
    .map((task) => (
```